Code/Tools/ProjectManager/Resources/ProjectManagerCompletionTags.txt

```diff
@@ -0,0 +1,46 @@
+AWS
```

Comment:
I'm not sure if we should be suggesting tags specific to any one company or product out of the box, same goes for some of the other tags (Barrier, Twitch, Synergy).

Reply:
I'll remove company and product specific tags. Waiting for response from UX before I address shrinking the tag list any further.